Gems/Atom/Tools/ShaderManagementConsole/Code/Source/ShaderManagementConsoleApplication.h

```diff
@@ -23,20 +23,15 @@
 #include <Atom/Window/ShaderManagementConsoleWindowNotificationBus.h>
 
 #include <AzQtComponents/Application/AzQtApplication.h>
+#include <AtomToolsFramework/Application/AtomToolsApplication.h>
```

Comment:
You may clean up some includes here since AtomToolsApplication.h already includes most of them